vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/implementation/VLogReasoner.java

```diff
@@ -808,4 +811,34 @@ ReasonerState getReasonerState() {
 	void setReasonerState(ReasonerState reasonerState) {
 		this.reasonerState = reasonerState;
 	}
+
+	@Override
+	public void getKbFacts(OutputStream stream) throws IOException {

```

Comment:
Method behaviour is inconsistent with its javadoc: the facts explicitly added in the KB may not appear in the rule heads, therefore method writes only the inferred facts.